### ISSUE TITLE
Random crashing when using other programs

### ISSUE THREAD

**Issue by gilmanjo** [SARL-Engineering/SideviewController#1]
When recording, there's a fatal crash occurring randomly, without warning, and without error message.  Happened once while trying to open Windows Task Manager while recording and another time when trying to start recording while Windows Task Manager was already open.

May be a hardware issue.  Have no idea.

**Comment by AaronRito** [comment]
Are you trying to access the same file/folder from separate threads?

**Comment by gilmanjo** [comment]
No, as far as I've been able to determine all functions being used by generated threads don't access QObjects or write data immediately accessible to other threads and are for all other purposes, threadsafe.  Only happens with FLIR/PT Grey camera, standard USB cameras doesn't have any issues.
